webview/src/components/breakpoint-button.svelte

```diff
@@ -1,27 +1,27 @@
 <script lang="ts">
   import type { WorkflowTask } from "../lib"
+  import { CircleFilled } from "svelte-codicons"
+
   export let workflowTask: WorkflowTask
-  let icon: string
   let hasBreakpoint = workflowTask.hasBreakpoint
-  $: {
-    hasBreakpoint
-    icon = hasBreakpoint ? "🔴" : "⚪"
-  }
 
   function toggleBreakpoint(workflowTask: WorkflowTask) {
     console.log("toggleBreakpoint", workflowTask)
     hasBreakpoint = workflowTask.hasBreakpoint = !workflowTask.hasBreakpoint
   }
 </script>
 
-<input type="button" value={icon} on:click={toggleBreakpoint.bind(undefined, workflowTask)} />
+<div
+  class="checkbox"
+  role="checkbox"
+  aria-checked={hasBreakpoint}
+  tabindex="0"
+  on:click={toggleBreakpoint.bind(undefined, workflowTask)}>
+  <CircleFilled fill={hasBreakpoint ? "#FF0000" : "currentColor"} />
+</div>
 
 <style>
-  input {
```

Comment:
This makes the corners look weird, they were round before.

Oh and can you please make this toggle with the spacebar?

<img width="35" alt="image" src="https://user-images.githubusercontent.com/52304/193156985-1f2e24b2-cea3-4962-820e-901efec3aa67.png">

Reply:
It used to work with the spacebar, I just checked.

Reply:
Using `Space` and `Enter` should now work. Since I'm using an icon (from [here](https://microsoft.github.io/vscode-codicons/dist/codicon.html)) for the circle now instead of passing in a value to an input, the focus outline is of the svg itself so it will be a rectangle instead of a circle.

I've made it more square 👇 but in order to get it round we would need to make changes to the icon itself.

<img width="149" alt="Screen Shot 2022-09-30 at 11 26 03 AM" src="https://user-images.githubusercontent.com/15069288/193324538-83c26155-84d8-4431-92bb-4ae7ef7d59b3.png">

Reply:
lgtm